tests/unit/Model/Command/UserMergeTest.php

```diff
@@ -12,12 +12,13 @@ public function shouldGenerateCorrectSignature(): void
     {
         $sourceUserId = 'source-user';
         $targetUserId = 'target-user';
+        $timestamp = time();
```

Comment:
Maybe use some static value instead of `time()`, which is the same as the default one -  to make sure the setters actually sets the value. This test would pass even it the timestamp value would not be given to the constructors.